types/testingutils/beacon_node.go

```diff
@@ -101,12 +103,12 @@ var Transactions = func() []bellatrix.Transaction {
 	return res.Transactions
 }()
 
-var TestingBeaconBlock = &bellatrix.BeaconBlock{
+var TestingBeaconBlock = &deneb.BeaconBlock{
```

Comment:
This section of code appears to be unused now. It can be safely deleted as we have transitioned to using the `go-eth2-client` blocks, as specified in our `beacon_node_consts.go`. For reference, see [beacon_node_consts](https://github.com/bloxapp/ssv-spec/blob/3d553a7de7b6ddeaf46bcc3a6d86258e20ae2730/types/testingutils/beacon_node_consts.go).